bindings/cs/rl.net.native/rl.net.live_model.h

```diff
@@ -47,6 +47,9 @@ extern "C" {
     API int LiveModelReportActionTaken(livemodel_context_t* livemodel, const char * event_id, reinforcement_learning::api_status* status = nullptr);
     API int LiveModelReportOutcomeF(livemodel_context_t* livemodel, const char * event_id, float outcome, reinforcement_learning::api_status* status = nullptr);
     API int LiveModelReportOutcomeJson(livemodel_context_t* livemodel, const char * event_id, const char * outcomeJson, reinforcement_learning::api_status* status = nullptr);
+	API int LiveModelReportOutcomeSlotF(livemodel_context_t* context, const char* event_id, int slot_index, float outcome, reinforcement_learning::api_status* status = nullptr);
```

Comment:
Why name this F instead of (I assume) Float?

Reply:
LiveModelReportOutcomeF already existed, so I stuck to the same naming convention.